packages/page-header/src/PageHeader.js

```diff
@@ -165,6 +167,7 @@ PageHeader.propTypes = {
   clientId: PropTypes.string,
   iconSrc: PropTypes.string,
   iconAlt: PropTypes.string,
+  variables: PropTypes.object,
```

Comment:
Can you add this to the PageHeaderProps type as well please? The type should be `Record<string, unknown>`

Reply:
Also, what do you think about naming the prop `spacesVariables`? That way it will be less generic